doc/source/starting.rst

```diff
@@ -84,9 +91,10 @@ By default if the file exists, it will not be overwritten and an error will be r
 This can be changed by ``overwrite=True``.
 
 .. warning::
-   Be careful with overwriting the original file (and ``overwrite=True`` in general) when writing a modified file out.
-   This will wipe out the original version, and if you have no version control,
-   may lead to losing information.
+   Overwriting the original file (with ``overwrite=True``) when writing a modified file out is discouraged.
+   This is because if your script using MontePy is buggy you have no real way to debug,
+   and recover from the issue if your original file has been been modified.
+
```

Comment:
We could add something here reminding users they can easily add a timestamp or UUID so that they don't have to delete or overwrite the output file every time